architecture/core_concepts/routes.adoc

```diff
@@ -37,30 +42,6 @@ Configuration] guide for information on deploying a router.
 ====
 endif::[]
 
-OpenShift routers provide external host name mapping and load balancing
-to link:pods_and_services.html#services[services] over protocols that
-pass distinguishing information directly to the router; the host name
-must be present in the protocol in order for the router to determine
-where to send it.
```

Comment:
Can you make sure this section isn't deleted?  It's useful preamble for the next paragraph that you moved.